### PR TITLE
xmedcon : proposals for nix systems without gnome

### PR DESCRIPTION
**Pull request by enlf-dev**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by enlf-dev** [comment]
Hi,

The wrapGAppsHook seems needed on systems without gnome. Otherwise gui crashes (see also issue [16285](https://github.com/NixOS/nixpkgs/issues/16285))

The homepage is set to the main url.

A shell hook is added to get those required schemas in a nix-shell.

The hicolor-icon-theme is added due to warnings of not finding icon(s) (optional).

Greetings,
Erik

File: pkgs/applications/science/medicine/xmedcon/default.nix [hunk 1]
```diff
@@ -24,11 +26,15 @@ stdenv.mkDerivation rec {
     zlib
   ];
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config wrapGAppsHook ];
+
+  shellHook = ''
```
Comment (flokli) [inline pkgs/applications/science/medicine/xmedcon/default.nix:31]: I don't think we set shellHook anywhere in nixpkgs, and I don't think we patch in random icon themes either. These are usually picked up ambiently from the environment, so I'd skip that part.

Comment (AndersonTorres) [inline pkgs/applications/science/medicine/xmedcon/default.nix:31]: Remove this.

**Comment by AndersonTorres** [comment]
Closing as abandoned.
Feel free to reopen.

**Pull request closed without merging**